client/src/pages/Edit.js

```diff
@@ -16,21 +17,14 @@ function Edit() {
 
   useEffect(() => {
     const editFoodId = async () => {
-      const response = await fetch(`http://localhost:5000/menu_items/${id}`); //change endpoint to hit 1 entry not all
-      const result = {
-        data: null,
-        error: null,
-      };
-
-      if (response.ok) {
-        result.data = await response.json();
-      } else {
-        result.error = await response.text();
-      }
-      console.log(result.data);
-      setEditFood(result.data);
+      fetch();
```

Comment:
What's this little guy?  Is it safe to remove it?

```suggestion
```

Reply:
I'm digging the scope increase - working on it today

Reply:
💪